src/components/elements/CommonDetailGrid.tsx

```diff
@@ -1,12 +1,15 @@
 import { Grid, Typography } from '@mui/material';
 
 import React, { ReactNode, useId } from 'react';
+import { useIsMobile } from '@/hooks/useIsMobile';
 
 // Grid of Key/Value pairs that has the specific styling used for Enrollment Details and custom client attributes
 
 export const CommonDetailGridContainer: React.FC<{
   children: ReactNode;
 }> = ({ children }) => {
+  const isTiny = useIsMobile('sm');
```

Comment:
I think we want to use`md` here to match the behavior of the `CommonDetailGridItem`s. It looks like there is a point between large and small where the table wraps, but the borders remain:

<img width="685" alt="Screenshot 2024-03-26 at 2 50 31 PM" src="https://github.com/greenriver/hmis-frontend/assets/5333982/20f84a0d-03a3-4e0c-a38f-7e282f649597">